assets/js/index.js

```diff
@@ -1,26 +1,173 @@
-const burger = document.querySelector('.header__container__parhead__nav__ul__responsive');
-const mobileNav = document.querySelector('.iwq');
+// form validation
+
+const details = [
+  {
+    id: 1,
+    title: 'Tonic',
+    list: {
+      list1: 'CANOPY',
+      list2: 'Back End Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/three.png',
+    desc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    lang: {
+      lang1: 'html',
+      lang2: 'css',
+      lang3: 'js',
+      lang4: 'ruby',
+    },
+    category: 'project1',
+  },
```

Comment:
- [ ] I noticed that although you created a JavaScript array to hold the contents of the projects, the hardcoded HTML contents for the projects list were not removed. :thinking: 
- [ ] According to the project requirements, the projects section needs to be created dynamically using information from the projects array. This means that the existing HTML code for the projects list needs to be removed, and the list should be added via JavaScript when the page loads. :100: 
- [ ]  Please make the necessary changes to ensure that the projects section is created dynamically, as shown in the attached screenshot of the project requirements. :+1: 

![image](https://user-images.githubusercontent.com/102699958/231513024-d2bd2106-4953-4ce7-9436-9ec29140495c.png)
- You can check an example of this student's project: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95